api/host_test.go

```diff
@@ -61,7 +62,7 @@ func TestIntegrationHosting(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	expRevenue := "15928888888855757473"
+	expRevenue := "15307662222190387473"
```

Comment:
I don't know why this changed, but I'm guessing some of the buffering constants were edited or something